tests/lib/options.js

```diff
@@ -356,12 +356,6 @@ describe("options", function() {
 
             assert.equal(currentOptions.parser, "test");
         });
-
-        it("should return a espree if --parser is not passed", function() {
```

Comment:
Sorry, I dint understand what you are trying to say here?
We always default to `espree` if the parser is not defined.. https://github.com/eslint/eslint/blob/master/lib/eslint.js#L435